pkgs/tools/misc/ix/default.nix

```diff
@@ -1,26 +1,31 @@
-{ lib, stdenv, fetchurl, makeWrapper, curl }:
+{ lib, resholve, fetchurl, runtimeShell, curl }:
 
-stdenv.mkDerivation {
+resholve.mkDerivation {
   pname = "ix";
   version = "20190815";
 
   src = fetchurl {
     url = "http://ix.io/client";
-    sha256 =  "0xc2s4s1aq143zz8lgkq5k25dpf049dw253qxiav5k7d7qvzzy57";
+    hash = "sha256-p/j/Nz7tzLJV7HgUwVsiwN1WxCx4Por+HyRgFTTRgnU=";
   };
 
-  nativeBuildInputs = [ makeWrapper ];
-
   dontUnpack = true;
 
   installPhase = ''
-    install -Dm +x $src $out/bin/ix
-  '';
+    runHook preInstall
+
+    install -Dm555 $src $out/bin/ix
 
-  postFixup = ''
-    wrapProgram $out/bin/ix --prefix PATH : "${lib.makeBinPath [ curl ]}"
+    runHook postInstall
   '';
 
+  solutions.default = {
+    scripts = [ "bin/ix" ];
+    interpreter = runtimeShell;
+    inputs = [ curl ];
+    keep."$echo" = true;
```

Comment:
It looks like this is solving for this line in the target script:

```bash
            d) $echo curl $opts -X DELETE ix.io/$OPTARG; return;;
```

I don't see an `echo` variable defined anywhere in the script, so I wonder if this is a bug/typo in the source? @peterhoeg @asymmetric have either of you used this `-i` flag?

Reply:
I've patched it out.